tools/nosimage/example/generate_example.go

```diff
@@ -95,31 +95,56 @@ func generateExample(filepath string, invalid bool) error {
 			}},
 		},
 		Ocrpcs: &rpb.OCRPCs{
-			OcProtocols: map[string]*rpb.OCProtocol{
-				"gnmi.gNMI": {
-					Version: "0.10.0",
-					MethodName: []string{
-						"Set",
-						"Subscribe",
-					},
-				},
-				"gnoi.healthz.Healthz": {
-					Version: "1.3.0",
-					MethodName: []string{
-						"Get",
-						"List",
-						"Acknowledge",
-						"Artifact",
-						"Check",
-					},
-				},
-				"gnoi.bgp.BGP": {
-					Version: "0.1.0",
-					MethodName: []string{
-						"ClearBGPNeighbor",
-					},
-				},
-			},
+			OcProtocols: func() map[string]*rpb.OCProtocol {
+				if valid {
+					return map[string]*rpb.OCProtocol{
+						"gnmi": {
+							Version: "0.10.0",
+							MethodName: []string{
+								"gnmi.gNMI.Set",
+								"gnmi.gNMI.Subscribe",
+							},
+						},
+						"gnoi": {
+							Version: "0.3.0",
+							MethodName: []string{
+								"gnoi.healthz.Healthz.Get",
+								"gnoi.healthz.Healthz.List",
+								"gnoi.healthz.Healthz.Acknowledge",
+								"gnoi.healthz.Healthz.Artifact",
+								"gnoi.healthz.Healthz.Check",
+								"gnoi.bgp.BGP.ClearBGPNeighbor",
+							},
+						},
+					}
+				} else {
```

Comment:
Please fix static check, "if block ends with a return statement, so drop this else and outdent its block"

Reply:
Done.